### ISSUE TITLE
kms 

### ISSUE THREAD

**Issue by ghost** [yuliskov/padavan-tweaks#1]
Добрый день. 
Хочу добавить в прошивку только kms. 
Как собрать только с этим пакетом я разобрался. 
Насколько я понял можно добавить его настройку в веб-интерфейс, но как это реализовать понять не могу.  Подскажите, пожалуйста.

**Comment by yuliskov** [comment]
@MyTimeKill дык ему особая настройка и не нужна. Для запуска можно добавить буквально одну строчку в разделе Скрипты>Выполнить после полного запуска маршрутизатора
```
# Windows 8/10 KMS Server
vlmcsd
```

**Comment by ghost** [comment]
Да это понятно, просто видел китайский форк и они там сделали управление c веб-ннтерфейса. 
Очень удобно, давно ищу как прикрутить. 
Вот здесь можно посмотреть, что они наворотили:
http://pdv.80x86.io/Advanced_Extensions_VLMCSD.asp

**Comment by yuliskov** [comment]
@MyTimeKill интересная штука. С удовольствием бы покопался в исходниках. Не в курсе, где они могут лежать?

**Comment by ghost** [comment]
Вот тема чела который это пилит, на китайском. 
http://www.right.com.cn/forum/thread-161324-1-1.html
У него там имеются ссылки на скрипты, не очень для меня понятные. 
http://opt.cn2qq.com/
https://github.com/hiboyhiboy/opt-script

Хотя не уверен, что это точно оно. 
Читал, что пилит он.

**Comment by yuliskov** [comment]
@MyTimeKill да, я про эту репу в курсе. Свой скрипт как-бы тоже брал отсюда. Но вот странность - в упор не вижу где там файлы веб-интерфейса.

**Comment by ghost** [comment]
Да нету, накатил прошивку от этого hiboy. 
Только вот и исходников этого мода по ссылке тоже нет, одни  бинарники.

**Comment by yuliskov** [comment]
@MyTimeKill [здесь](https://github.com/hiboyhiboy/opt-file) у него лежат файлы от админки и остальная ерунда. Вроде бы то что нужно, но файлов от kms там по прежнему нет.
